core/services/src/main/java/org/exoplatform/services/cms/impl/ImageUtils.java

```diff
@@ -73,9 +73,15 @@ public static InputStream scaleImage(BufferedImage image, int maxWidth, int maxH
         //with expected height.
         thumbImage = Scalr.resize(image, Scalr.Mode.FIT_TO_HEIGHT, maxHeight);
       } else {
-        //BALANCED: Used to indicate that the scaling implementation should use a scaling operation balanced 
-        //between SPEED and QUALITY
-        thumbImage = Scalr.resize(image, Scalr.Method.BALANCED, maxWidth, maxHeight);      
+        int originWidth = image.getWidth();
+        int originHeight = image.getHeight();
+        Scalr.Mode fitMode = Scalr.Mode.AUTOMATIC;
+        if (originWidth > originHeight) {
+          fitMode = Scalr.Mode.FIT_TO_HEIGHT;
+        } else if (originHeight > originWidth) {
+          fitMode = Scalr.Mode.FIT_TO_WIDTH;
+        }
+        thumbImage = Scalr.resize(image, Scalr.Method.QUALITY, fitMode, maxWidth, maxHeight, Scalr.OP_ANTIALIAS);

```

Comment:
Should we replace ImageUtils call by a call to ImageResizeService ?

Reply:
Yes we can

Reply:
We should do it in the same task so

Reply:
Can you rebase the PR ?

Reply:
rebased